src/index.js

```diff
@@ -0,0 +1,216 @@
+        'prettier/prettier': [
+          'error',
+          {
+            singleQuote: true,
+            trailingComma: 'none',
+            bracketSpacing: true,
+            jsxBracketSameLine: true
+          }
+        ],
+        // Validate JSX has key prop when in array or iterator
+        'react/jsx-key': 'error',
+        // Prevent comments from being inserted as text nodes
+        'react/jsx-no-comment-textnodes': 'error',
+        // Prevent usage of unsafe target="_blank", ensure anchors also have rel="noreferrer noopener"
+        'react/jsx-no-target-blank': 'error',
+        // Ensure JSX components are PascalCase
+        'react/jsx-pascal-case': 'error',
+        // Disable forced validation of React prop types
+        'react/prop-types': 'off'
+      }
+    }
+  });
+
+  if (options.eslint) {
+    neutrino.use(loaderMerge('lint', 'eslint'), options.eslint);
+  }
+
+  if (options.cssModules) {
+    neutrino.config.module
+      .rule('plain-style')
+      .test(/\.css$/)
+      .include.add(neutrino.options.node_modules)
+      .end()
+      .use('style')
+      .loader(require.resolve('style-loader'))
+      .end()
+      .use('css')
+      .loader(require.resolve('css-loader'));
+
+    neutrino.config.module
+      .rule('style')
+      .exclude.add(neutrino.options.node_modules)
+      .end()
+      .use('css')
+      .options({ modules: true });
+  }
+
+  if (!options.react) {
+    Object.assign(options, { react: {} });
+  }
+
+  const reactOptions =
+    options.library || options.components
+      ? Object.assign(options.react, {
+          html: options.components ? { title: 'React Preview' } : undefined,
+          babel: merge(options.react.babel || {}, {
+            presets: [
+              [
+                'babel-preset-env',
+                {
+                  targets: {
+                    browsers: [
+                      'last 1 Chrome versions',
+                      'last 1 Firefox versions',
+                      'last 1 Edge versions',
+                      'last 1 Safari versions',
+                      'last 1 iOS versions'
+                    ]
+                  }
+                }
+              ]
+            ]
+          })
+        })
+      : options.react;
+
+  neutrino.use(react, reactOptions);
+
+  if (options.envs) {
+    neutrino.config.plugin('env').tap(envs => [...envs, ...options.envs]);
+  }
+
+  neutrino.config.module
+    .rule('compile')
+    .use('cache')
+    .before('babel')
+    .loader(require.resolve('cache-loader'));
+
+  if (options.components) {
+    const components = join(neutrino.options.source, 'components');
+
+    neutrino.config.when(
+      neutrino.options.command === 'start',
+      () => {
+        neutrino.options.entry = 'stories'; // eslint-disable-line no-param-reassign
+      },
+      () => {
+        neutrino.config.entryPoints.delete('index');
+        readdirSync(components).map(component =>
```

Comment:
This seems like another use-case for Neutrino having multi-entrypoint support. I've been meaning to file a Neutrino issue with a proposal for a Treeherder use case that would also simplify this.